Rocket.Chat/Views/Cells/Chat/ChatMessageCell.swift

```diff
@@ -140,6 +157,7 @@ final class ChatMessageCell: UICollectionViewCell {
         avatarView.prepareForReuse()
 
         for view in mediaViews.arrangedSubviews {
+            mediaViews.removeArrangedSubview(view)
```

Comment:
Not sure about this one, but I believe we should call it as well while preparing cell for reuse?

Reply:
Yes, you're right! Makes sense 👍